main.cpp

```diff
@@ -30,23 +32,46 @@ class Mesh {
 constexpr uint32_t MAX_KEY_LENGTH = 62;
 constexpr uint32_t MAX_ENTRIES_PER = 100;
 constexpr uint32_t MAX_TEST_ITERATIONS = 89;
-constexpr uint32_t INITIAL_SEED = 89;
+constexpr uint32_t INITIAL_SEED = 1713623895;
 
 std::string char_values_from{"0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"};
 
-bool pr_754_approach(std::unordered_map<std::string, std::vector<Mesh *> *> &bfxm_hash_table, const std::string& hash_name, const Mesh * thus)
+template<typename T> void remove_all_references_to(const T *thus, owner<std::vector<T *> *> &vec, bool& collection_was_deleted) {
```

Comment:
This function, and the one below it, are what I would like to start using in Vega Strike wherever applicable.

Reply:
And use a simplified version in cases where we don't need to worry about deleting the collection right then.

Reply:
Just included this code into Vega Strike. Done ✅